### ISSUE TITLE
JDK9: provide Automatic-Module-Name entry in manifest

### ISSUE THREAD

**Issue by zyxist** [remondis-it/remap#28]
Hello!

Java 9 introduced the module system. If a non-modular JAR is used in a modular app, Java creates a so-called automatic module from it, whose name is derived from the JAR name (by default).

It is very important for modules to have stable names (just like packages), because changing it later may lead to the compilation errors and module hell. If you don't want to migrate to Java 9 yet, you can add the following entry to the MANIFEST file for forward compatibility:

```java
Automatic-Module-Name: com.remondis.remap
```

It is neutral for older Java versions, and Java 9 will use it as the name of the automatic module, so that I could do the following thing in my application, without worrying about the future:

```java
module com.example.foo {
   requires com.remondis.remap;
}
```

(note: the official recommendation is to name the module from the root package, and use reverse-DNS style).

**Comment by schuettec** [comment]
Thank your for that detailed advice!

It is definitely our plan to support Java 9 so I will give that a try.

**Comment by schuettec** [comment]
We decided to make ReMap forward compatible by adding the suggested `Automatic-Module-Name`.
The manifest attribute will be available in version `2.0.1`. 

Thanks for creating an issue!
